app/views/spaces/show/_where.html.erb

```diff
@@ -7,6 +7,11 @@
       <%= @space.post_number %>
       <%= @space.post_address %>
     </span>
+  </div>
+   <div class="flex gap-1 mb-3">
```

Comment:
Denne diven trengs ikke finnes om @space.location_description er blank, så fex test for:

```ruby
 if @space.location_description.present?
```